src/versionbits.cpp

```diff
@@ -30,8 +36,8 @@ ThresholdState AbstractThresholdConditionChecker::GetStateFor(const CBlockIndex*
             cache[pindexPrev] = ThresholdState::DEFINED;
             break;
         }
-        if (pindexPrev->GetMedianTimePast() < nTimeStart) {
-            // Optimization: don't recompute down further, as we know every earlier block will be before the start time
+        if (pindexPrev->nHeight + 1 < height_start) {
+            // Optimization: don't recompute down further, as we know every earlier block will be before the start height
```

Comment:
there's not really a great way to rewrite this, but can we comment why it is +1

Reply:
how it is doc'd below is great

Reply:
Done